README.md

```diff
@@ -126,30 +133,36 @@ On first start, you will likely see an error stating that there are no mail acco
 
 You can then proceed by creating the postmaster alias and by creating DKIM keys.
 
-``` BASH
-docker-compose up -d mailserver
-
-# you may add some more users
-# for SELinux, use -Z
-./setup.sh [-Z] email add <user@domain> [<password>]
-
-# and configure aliases, DKIM and more
-./setup.sh [-Z] alias add postmaster@<domain> <user@domain>
-./setup.sh [-Z] config dkim
-```
+``` CONSOLE
+$ docker-compose up -d mailserver
 
-In case you're using LDAP, the setup looks a bit different as you do not add user accounts directly. Postfix doesn't know your domain(s) and you need to provide it when configuring DKIM:
+$ # you may add some more users
+$ # for SELinux, use -Z
```

Comment:
This is a bit weird, in that while it's not output from the previous command, it's a bit weird annotating as a command with `$` as well (which is included in the rendered markdown still).

This also occurs elsewhere in the fenced blocks that have switched to `CONSOLE`.

Reply:
I agree. Will see what I can do